teitelbot/util/string_utils.py

```diff
@@ -35,3 +35,7 @@ def replace_words(words: list, replacement_map: dict) -> list:
 
 def translate_to_heb(string_in_english: str) -> str:
     return translator.translate(string_in_english, src='iw', dest='en').text
+
+
+def is_true(string: str) -> bool:
```

Comment:
Ew...

Reply:
what do you suggest?

Reply:
This is probably fine, just seems strange.
Is it really necessary? Is it some kind of convention?

Reply:
used in config parsing.
export a=false
considered as true boolean.

Reply:
Aight